src/convertToObject.js

```diff
@@ -6,7 +6,90 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const object = {};
+
+  const splitString = sourceString
+    .split(';');
```

Comment:
The `split(';')` method will create an array of properties, but it will also include empty strings for trailing semicolons. Consider filtering out empty strings after splitting to avoid processing empty properties.